### DISCUSSION TITLE
TGBlazor

### DISCUSSION THREAD

**Discussion by frezeen** [DamianMorozov/OpenTgResearcher#64] [General]
is tgdownloader blazor a project closed?

i think webapp+docker support  is the way . its universal, u can open it on lots OS and Systems

**Comment by DamianMorozov** [comment]
Support for console and docker versions exists.
There is not enough time to support Blazor version.
In the near future all efforts are focused on the Desktop version for Windows.

**Comment by DamianMorozov** [comment]
No time for the web version.
